src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/TotalRecognizedDelegatesCard.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../GenericDelegateCard';
```

Comment:
you should rename the default export here too

Reply:
done